### PR TITLE
feat: add devDependencies install: (#40)

### LINKED ISSUE #40: 实现 package.json 的 devDependencies 依赖安装

### PR DESCRIPTION
**Pull request by qq1051766345**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/xun082/create-neat/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## What is the current behavior?

- Closes #40 
- Related to #24 

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by uaenaTzx** [comment]
关联这个 issue：https://github.com/xun082/create-neat/issues/24

**Comment by uaenaTzx** [comment]
把分支名规范一下，feat: add devDependencies install: (#40 )

File: packages/core/src/utils/dependenciesInstall.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import fs from "fs";
+import { spawn } from "child_process";
+import path from "path";
+/**
```
[the remark this answers is not in the document]
Comment (uaenaTzx) [reply packages/core/src/utils/dependenciesInstall.ts:5]: @qq1051766345 看得到吗

**Review by uaenaTzx — APPROVED** [review]

File: packages/core/src/utils/dependenciesInstall.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (qq1051766345) [reply packages/core/src/utils/dependenciesInstall.ts:5]: 现在看到了

File: packages/core/src/utils/dependenciesInstall.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    try {
+      const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, "utf-8"));
+      const devDependencies = packageJson.devDependencies;
+
+      if (devDependencies) {
+        console.log("Installing devDependencies...");
+        const devDepsArray = Object.entries(devDependencies).map(
+          ([dep, version]) => `${dep}@${version}`,
+        );
+        const pm = spawn(
```
[the remark this answers is not in the document]
Comment (qq1051766345) [reply packages/core/src/utils/dependenciesInstall.ts:39]: 已加

File: packages/core/src/utils/dependenciesInstall.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (qq1051766345) [reply packages/core/src/utils/dependenciesInstall.ts:5]: 已修改

**Pull request merged**